docs/Reboots.md

```diff
@@ -0,0 +1,29 @@
+# What causes reboots?
+
+Under certain conditions, MachineConfig updates will cause the Machine
+Config Daemon to reboot the nodes affected by the change. This document
+aims to identify exactly what MachineConfig changes cause a reboot, with
+the intent that any changes will not be unexpected.
+
+A MachineConfig change will cause a reboot under the following
+conditions:
+
+-   An OS update is being applied.
+-   Linux kernel arguments have changed.
+-   FIPS (Federal Information Processing Standards) configuration has
```

Comment:
FIPS alone is fine also just link here: https://github.com/openshift/machine-config-operator/blob/ca8fcb887f153604556f2706cee8a33165879797/docs/MachineConfiguration.md#fips

Reply:
can you also confirm FIPS is a day 2 change? I didnt think it was?

Reply:
FIPS is a day one operation -- it is not a toggable option.

```
Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
```

https://docs.openshift.com/container-platform/4.7/installing/installing-fips.html

Reply:
I didn't think it was either, but I'll confirm. I spelled it out because it was an unfamiliar (to me, anyway) acronym, but I'll add the link you mentioned.

Reply:
Thanks for the confirmation @darkmuggle! I'll make a mention of this.